packages/suspense/src/utils/warnInDev.ts

```diff
@@ -1,5 +1,7 @@
+import { isDevelopment } from "#is-development";
+
 export function warnInDev(expectedCondition: boolean, message: string) {
```

Comment:
note that with this pattern the `expectedCondition` is still being evaluated and thus it has to stay as part of the final bundle, if you'd abandon this pattern and just wrap `console.warn` with the `isDevelopment` check then you could end up with a slightly smaller prod bundle

Reply:
Yeah, that's a nice little bump.